src/sample.ts

```diff
@@ -102,6 +102,7 @@ export function sample(...args: ReadonlyArray<unknown>): unknown {
   return purry(sampleImplementation, args);
 }
 
+// eslint-disable-next-line max-statements -- TODO: Can we simplify or split this?
```

Comment:
**question:** i see this function has a lot of guards, which we don't really have in other functions. for example, for `chunk`, we don't check that `size` is a positive integer. if we eliminate this, that'd bring us under the limit right?

Reply:
I think you are right and it also looks like how the built-in functions chose to handle this: for example `[1,2,3].slice(1.5)`.

I'll remove them